engine/moduleManager/modules/WindowModule.cpp

```diff
@@ -0,0 +1,139 @@
+﻿//
+// Created by larsv on 12/11/2024.
+//
+
+#include "WindowModule.hpp"
+
+#include <cmath>
+#include <SDL.h>
+
+#include <iostream>
+
+#include "BlockyEngine.hpp"
+
+WindowModule::WindowModule() {
+	if (SDL_InitSubSystem(SDL_INIT_VIDEO) < 0) {
+		std::cerr << "Couldn't init video: " << SDL_GetError() << std::endl;
+		return;
+	}
+
+	//Create window
+	window = SDL_CreateWindow("SDLTest", SDL_WINDOWPOS_UNDEFINED, SDL_WINDOWPOS_UNDEFINED,
```

Comment:
This can become something like a window property if the module is initializing it.
```suggestion
	struct WindowProperties {
	  int width;
	  int height;
	  size_t options; //Bitfield eventueel
	  WindowProperties(int newWidth, int newHeight, size_t newOptions): width(newWidht), height(newHeight), 
          options(newOptions) {}
	};
```
En de window nog steeds een uptr, hierdoor hebben we geen statische constanten die je aanroept :)

Reply:
Oh dat is geen change die ik erin wil hebben, maar zoals ik dacht een suggestie zodat we de create window not volpompen met vaste waarden.

Reply:
Waarschijnlijk als WindowModule daadwerkelijk geïmplementeerd wordt